lib/ide-haskell-repl-view.coffee

```diff
@@ -196,6 +199,20 @@ class IdeHaskellReplView
   setPrompt: (prompt) ->
     @promptDiv.innerText = prompt + '>'
 
+  getCompletions: (prefix) =>
+    return [] if prefix == '' or prefix == ' '
+    @ghci.sendCompletionRequest prefix
```

Comment:
Also, should probably check the return value of `sendCompletionRequest`. If it's `false`, `onComplete` will not fire this time, and we'll be left with a dangling promise. Probably should return `[]` or reject the promise outright in this case.